lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -52,21 +68,25 @@ def indexables
       # having duplicates if BUNDLE_PATH is set to a folder inside the project structure
 
       # Add user specified patterns
-      indexables = @included_patterns.flat_map do |pattern|
+      patterns = indexable_included_file_patterns.sort.map! { |dir| File.join(dir, "*.rb") }
```

Comment:
Why do we need to sort?

Reply:
Good call. I was considering using a bsearch at one point, but ended up not. Removed in https://github.com/Shopify/ruby-lsp/pull/2082/commits/228b84fe4aa336ebce632054499bbce407ab2f40